src/main/java/team/asd/service/CalendarService.java

```diff
@@ -0,0 +1,109 @@
+package team.asd.service;
+
+import team.asd.constants.DateElement;
+import team.asd.exceptions.WrongArgumentException;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.time.temporal.WeekFields;
+import java.util.Locale;
+import java.util.Objects;
+
+public class CalendarService implements IsCalendarService {
+	@Override
+	public String toString(LocalDate date) {
+		if (Objects.isNull(date)) {
+			return null;
+		}
+
+		return date.format(getFormatter());
+	}
+
+	private DateTimeFormatter getFormatter() {
+		return DateTimeFormatter.ofPattern("yyyy-MM-dd");
```

Comment:
Move this String pattern to constants
Or you can even define DateTimeFormatter as a class variable